pkg/cmd/kafka/consumergroup/describe/describe.go

```diff
@@ -93,6 +94,8 @@ func NewDescribeConsumerGroupCommand(f *factory.Factory) *cobra.Command {
 		return cmdutil.FilterValidConsumerGroupIDs(f, toComplete)
 	})
 
+	flagutil.EnableStaticFlagCompletion(cmd, "output", flagutil.ValidOutputFormats)
```

Comment:
nitpick: this could be further simplified by having a specific util for "output". There is still repetition here and the values will likely always be the same for the value flag name and the valid values.

Something like `flagutil.RegisterOutputFlagCompletion()`

Reply:
Having a separate method for `--output` will indeed reduce repetition. I will suggest to have `RegisterOutputFlagCompletion` for output flag and `RegisterOutputFlagCompletion` for other flags, else we'll have to define a new method for every flag that will require static completion. What do you think?

Reply:
I don't think we should define one for every flag, but for the extremly common ones like this it is good.

Reply:
Kept the `EnableStaticFlagCompletion` with a separate method to enable output flag.